### DISCUSSION TITLE
unsatisfied with overall distance snap functionality in the editor

### DISCUSSION THREAD

**Discussion by sponnify** [ppy/osu#25116] [Ideas]
i’ve been testing out the editor for a while and have noticed some differences in the distance snap functionality compared to the stable release. in stable, holding down alt + scrollwheel changes the distance spacing, but in lazer, this shortcut changes the master volume instead. this happens with the mouse cursor anywhere on the screen, whereas in stable, it only happens when the mouse is in the bottom-right corner of the screen.

i’m not sure why this same system can’t be implemented in lazer—such as it only changing master volume when the mouse cursor is on the left—rather than adding ctrl to the shortcut. this change means that all mappers transitioning to lazer will have to restructure their muscle memory.

another point of concern is the relocation of distance snap to the right-side panel. most mappers are used to it being associated with the beat snap divisor on the top-right side. i’m not sure if this is the best approach.

im not sure how i feel about the difference in functionality of distance spacing in lazer compared to standard. may be a personal preference of mine, but im personally used to the object you want to place using distance spacing being "anchored" to where the playhead is in the timeline, rather than it being able to move freely to beats ahead or behind the playhead. i much rather the process in stable being more controlled.

i believe these points are worth considering for a smoother transition for mappers from stable to lazer. i'm interested hearing what other mappers have to say about this topic.

**Comment by peppy** [comment]
> im not sure how i feel about the difference in functionality of distance spacing in lazer compared to standard. may be a personal preference of mine, but im personally used to the object you want to place using distance spacing being "anchored" to where the playhead is in the timeline, rather than it being able to move freely to beats ahead or behind the playhead. i much rather the process in stable being more controlled.

You can change this from the menu.

**Reply by sponnify** [reply]
not sure why it isnt this by default though

**Reply by peppy** [reply]
Because to someone who isn't coming from stable, the new setting is likely more intuitive. At least it is for me.
